src/main/java/com/microsoft/azure/springcloudplayground/generator/ProjectGenerator.java

```diff
@@ -20,6 +20,8 @@
 import org.springframework.util.Assert;
 import org.springframework.util.FileSystemUtils;
 import org.springframework.util.StreamUtils;
+import org.springframework.util.StringUtils;
+import retrofit2.http.HEAD;
```

Comment:
Seems this `HEAD` not used?

Reply:
yeah，should remove this.

Reply:
It is somewhat wired from IDE, as I enabled `auto import`, should remove this automatically.